src/polyfem/solver/SolveData.cpp

```diff
@@ -164,14 +164,13 @@ namespace polyfem::solver
 
 			if (use_adaptive_barrier_stiffness)
 			{
-				contact_form->set_weight(1);
+				contact_form->set_barrier_stiffness(1);
```

Comment:
why the barrier stiffness is not the weight anymore?

Reply:
Because the contact form has two weights: the acceleration scale and the barrier stiffness. Both need to be set at different points in the code and if they are premultiplied it becomes easy to overwrite with only one and forget to do the multiplication.